R/pkg/R/functions.R

```diff
@@ -3951,7 +3951,6 @@ setMethod("map_values",
 #' @rdname column_collection_functions
 #' @aliases map_zip_with map_zip_with,characterOrColumn,characterOrColumn,function-method
 #'
-#' @examples
```

Comment:
this was throwing a warning from `roxygen2` about empty `@examples`.

should add an example, but just deleted for now